docs/09-Configuration reference/02-Custom Resource Definitions.md

```
@@ -2177,6 +2563,205 @@ GatewayHTTPAPI is the Schema for the HTTPAPIs API
+
+
+
+
+
+
+
+
+
+| Field | Description | Default | Validation |
+| --- | --- | --- | --- |
+| `endpoint` _string_ | Endpoint URL for the signal (e.g., "http://my-collector:4318", "grpc://my-collector:4317").<br />Supported schemes: http, https, grpc. Bare host:port is treated as HTTP.<br />Protocol is inferred from the URL scheme. HTTP/protobuf is the default for firewall compatibility. |  | MinLength: 1 <br />Pattern: `^(https?://|grpc://|[a-zA-Z0-9])` <br /> |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 4; Actual: 6; Too many cells, extra data will be missing

(MD056, table-column-count)